hardhat.config.ts

```diff
@@ -26,16 +27,55 @@ task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
   }
 });
 
-// You need to export an object to set up your config
-// Go to https://hardhat.org/config/ to learn more
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
+const config: HardhatUserConfig = {
+  solidity: "0.8.2",
```

Comment:
please bump it to 0.8.9 as it's the latest version supported by hardhat https://hardhat.org/reference/solidity-support.html#supported-versions

Reply:
done